event-system/src/subscribe.js

```diff
@@ -6,6 +6,7 @@ let router = express.Router();
 
 router.get('/subscribe/:namespace/:room', (req, res) => {
   let client = redis.createClient();
+  req.socket.setTimeout(Number.MAX_SAFE_INTEGER);
```

Comment:
so sending the keep-alive header tells the browser to keep the connection open. this line tells the server to keep the connection open. But it probably shouldn't be part of this PR...